### PR TITLE
Project examples/ runs resulting in errors (PR)

### PR DESCRIPTION
**Pull request by gsstoykov**
**Description**:

This PR fixes the following:
- Correctly running the examples on MacOS
- VSCode related files where still committed to the repository

**Related issue(s):**

**Fixes:** https://github.com/hashgraph/hedera-sdk-cpp/issues/546 

**Notes for reviewer**:

The added run_examples script is created with the purpose of having an easier way to identify if there is an overall problem with the example run flows.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by rwalworth — APPROVED** [review]
LGTM

File: sdk/main/src/impl/Network.cc [hunk 1]
```diff
@@ -189,7 +190,8 @@ NodeAddressBook Network::getAddressBookForLedgerId(const LedgerId& ledgerId)
     return {};
   }
 
-  std::ifstream infile(ledgerId.toString() + ".pb", std::ios_base::binary);
+  std::string buildPath = std::filesystem::current_path().string() + "/addressbook/" + ledgerId.toString()+".pb";
```
Comment (rwalworth) [inline sdk/main/src/impl/Network.cc:193]: This actually breaks running examples from the `build` folder

**Review by rwalworth — CHANGES_REQUESTED** [review]

File: scripts/run_examples.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+fi
+
+# Iterate through the executables in the directory
+for exe in "$exe_dir"/*; do
+    if [ -x "$exe" ]; then
+        echo "Running $exe with arguments: $arg1 $arg2"
+        "$exe" "$arg1" "$arg2"
+        echo "Finished running $exe"
+    fi
+done
```
Comment (rwalworth) [inline scripts/run_examples.sh:23]: Is there a way to add being able to just run executable? Some of the examples never finish (end with a `while (true) {}` loop) so this won't run through all examples anyways

Comment (rwalworth) [reply scripts/run_examples.sh:23]: I might recommend installing this script as well, so anybody who downloads a release can run examples the same way

Comment (rwalworth) [reply scripts/run_examples.sh:23]: Also, not all examples run with the same two input arguments (see `ConstructClientExample` and `GeneratePrivateKeyFromMnemonicExample`). Should these maybe be included in a separate script? Or possibly a script made for each example?

Comment (gsstoykov) [reply scripts/run_examples.sh]: Will move the script topic to a separate discussion and issue.

**Review by deyanzz — APPROVED** [review]
LGTM.

**Pull request merged**